awx/ui_next/src/screens/Inventory/InventoryGroups/InventoryGroups.jsx

```diff
@@ -1,10 +1,42 @@
-import React, { Component } from 'react';
-import { CardBody } from '@patternfly/react-core';
+import React from 'react';
+import { withI18n } from '@lingui/react';
 
-class InventoryGroups extends Component {
-  render() {
-    return <CardBody>Coming soon :)</CardBody>;
-  }
+import { Switch, Route, withRouter } from 'react-router-dom';
+
+import InventoryGroupAdd from './InventoryGroupAdd/InventoryGroupAdd';
+
+import InventoryGroup from './InventoryGroup/InventoryGroup';
+
+function InventoryGroups({ setBreadcrumb, inventory }) {
+  return (
+    <Switch>
+      {[
```

Comment:
Do we need to wrap the Routes in brackets?

Reply:
no we don't